adventure_game/main.py

```diff
@@ -1,6 +1,8 @@
 import time 
 import random 
 from Ellen_input import valid_input
+from Joe_input import Y 
+from Joe_input import N 
```

Comment:
Cool that you're using inputs, you can also import both Y & N inputs on the same line like so:
`from Joe_input import Y, N`